platform/platform-manager.js

```diff
@@ -0,0 +1,97 @@
+const { fork } = require('child_process');
+const fs = require("fs-extra");
+let ipc = require('node-ipc');
+
+let services = {
+  "app-manager": {
+    path: __dirname + "/app-manager/app-manager.js",
+    socket: undefined,
+    process: undefined
+  },
+  "api-server": {
+    path: __dirname + "/api-server/server.js",
+    socket: undefined,
+    process: undefined
+  }
+};
+
+// data = {
+//   "meta": {
+//     "recipient": "app-manager",
+//     "event": "app-deployment",
+//     "sender": "api-server",
+//   },
+//   "payload": {}
+// }
```

Comment:
Write a comment statement in the beginning as "data is of the format as described". To make it more readable.
Also, use multiline comments